src/css/common/pp-semantic-ui-overrides.scss

```diff
@@ -17,3 +17,7 @@
 #{$pp-scope}.ui.popup.pp-popup-long-line {
   max-width: 420px;
 }
+
+.ui.page.modals.dimmer.transition.visible.active {
+  z-index: 10000050;
```

Comment:
I guess we could use our variables just as usual :) Importing them is ok, as long as we don't generate too much unnecessary CSS (and we don't, when we import non-CSS entities like mixins or variables)